06-Education_training.Rmd

```diff
@@ -302,6 +302,13 @@ to compensate for the time required to participate. In addition, to increase par
 from a diverse community, URSSI will provide travel support for a small number of workshop 
 participants, determined based on need in individual cases. 
 
+6. _Packaging and Sharing_
+As URSSI develops content related to the educational activities described in this chapter, we
+will package that content into Carpentries-style lessons. We will make the content freely 
```

Comment:
There is no specific Carpentries format. And we're also not striving to make these into 4 hour modules. So I'd skip this since it is not a thing.

Reply:
I modified the text to say that we will package them in a shareable format.